recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston_%.bbappend

```diff
@@ -13,13 +13,20 @@ python __anonymous () {
 }
 
 SRC_URI_append = "file://weston-seats.rules \
+                  file://weston-seats-kf.rules \
 "
 
 FILES_${PN} += " \
     ${sysconfdir}/udev/rules.d/weston-seats.rules \
+    ${sysconfdir}/udev/rules.d/weston-seats-kf.rules \
```

Comment:
this is not needed as we do not install "*kf*" version of the file

Reply:
Done